recipes/whisper-cpp/all/conanfile.py

```diff
@@ -114,13 +152,19 @@ def generate(self):
         if self.options.no_f16c:
             tc.variables["WHISPER_NO_F16C"] = True
 
+        if self.options.get_safe("with_openvino"):
+            tc.variables["WHISPER_OPENVINO"] = True
+            tc.variables["CMAKE_CXX_STANDARD"] = self.settings.get_safe("compiler.cppstd", 11)
```

Comment:
This is wrong because it tries to set `CMAKE_CXX_STANDARD` to `17gnu` for example.